levels/utils.go

```diff
@@ -12,9 +16,25 @@ func CmdOK(cmd string) bool {
 	return err == nil
 }
 
-func print_line(text string) {
+var key_pressed = false
+
+func print_line(text string, keypress chan []byte, echo_state bool) {
+	var counter = 0
+	var b []byte = make([]byte, 1)
 	for _, char := range text {
 		print(string(char))
+		if echo_state == false {
+			counter++
+			go func() { os.Stdin.Read(b); keypress <- b }()
+			select {
+			case <-keypress:
```

Comment:
@Foxiy @pepol if I am not mistaken, a simple check for correct ASCII values here should be sufficient in order to prevent keys other than SPACE and ENTER to skip pretty printing.